pandas/core/indexes/category.py

```diff
@@ -293,6 +293,15 @@ def values(self):
         """ return the underlying data, which is a Categorical """
         return self._data
 
+    @property
+    def itemsize(self):
+        return self.values.itemsize
```

Comment:
let's add a comment why you are overwriting this here (default is to use `_ndarray_values`, which are codes, but Categorical defines itemsize as the itemsize of the categories). 
Also, would it be more consistent to do `self._values.itemsize`? (not that it would differ in this case)

Reply:
More consistent with the parent's implementation, yes, but I'd prefer to use the public API where possible.

Reply:
> , but I'd prefer to use the public API where possible.

But the public api is not meant to be consistent, but to maintain backwards compatibility :-) (at least for now)